fendermint/vm/actor_interface/src/system.rs

```diff
@@ -2,9 +2,23 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 use cid::Cid;
 use fvm_ipld_encoding::tuple::*;
+use fvm_shared::address::Address;
+use lazy_static::lazy_static;
+
+use crate::eam::EthAddress;
 
 define_singleton!(SYSTEM { id: 0, code_id: 1 });
 
+lazy_static! {
+    /// The Ethereum null-address 0x00..00 can also be used to identify the system actor.
```

Comment:
@raulk, is this also the case in lotus?

Reply:
Based on [this](https://github.com/filecoin-project/lotus/blob/fbafe1c507f362c82d2cd5540b6c5d580dae8e74/chain/gen/genesis/genesis_eth.go#L25-L29) I think they just create Ethereum account actors with these addresses, they aren't the same as the system address. It is only me who thought since we have this notion of sending these calls as the system actor, and that's actually what Lotus does too, then why not make it official?